silkworm/silkrpc/ethdb/listener_cbor_log.h

```diff
@@ -0,0 +1,67 @@
+
+/*
+   Copyright 2023 The Silkworm Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+#pragma once
+
+#include <vector>
+
+#include <silkworm/silkrpc/common/log.hpp>
+#include <silkworm/silkrpc/json/types.hpp>
+
+#include <cbor/listener.h>
+
+#define UNUSED(expr) (void)(expr)
```

Comment:
Please avoid using macro except for exceptional situations (e.g. logging macros)

Reply:
ok